packages/squiggle-lang/src/public/SqProject/ProjectItem.ts

```diff
@@ -225,13 +234,28 @@ export class ProjectItem {
         evaluate: asyncEvaluate,
       });
 
-      const bindings = contextAfterEvaluation.stack.asBindings();
       const exportNames = new Set(expression.value.value.exports);
-      const exports = bindings.filter((_, key) => exportNames.has(key));
+      const bindings = contextAfterEvaluation.stack
+        .asBindings()
+        .mapEntries(([key, value]) => {
+          let _value = value;
+          if (exportNames.has(key)) {
+            _value = value.mergeTags({
+              exportData: { sourceId: this.sourceId, path: [key] },
+            });
+          }
+          return [key, _value];
+        });
+      const exports = bindings.filter(
+        (value, key) => value.tags?.exportData !== undefined
```

Comment:
Somehow this leads to tagged unexported values showing in "Exports":
<img width="719" alt="Screenshot 2024-01-17 at 14 24 01" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/1ab94e24-5a95-45e5-b8e5-c91284a18ab6">

Reply:
(Fixed! The problem is that it should have been ``.exportsData()``, not ``.exportData``.